includes/class-notices.php

```diff
@@ -619,18 +620,41 @@ public static function print_frontend_notice( $message, $echo = true, $notice_ty
 		 */
 		$notice_args = apply_filters( 'give_frontend_notice_args', $notice_args );
 
-		// Note: we will remove give_errors class in future.
-		$error = sprintf(
-			'<div class="give_notices give_errors" id="give_error_%1$s">
-				<p class="give_error give_notice give_%1$s" data-dismissible="%2$s" data-dismiss-interval="%3$d">
-					%4$s
-				</p>
-			</div>',
-			$notice_type,
-			give_clean( $notice_args['dismissible'] ),
-			absint( $notice_args['dismiss_interval'] ),
-			$message
-		);
+		switch ( $notice_args['dismiss_type'] ) {
+			case 'auto':
+				// Note: we will remove give_errors class in future.
+				$error = sprintf(
+					'<div class="give_notices give_errors" id="give_error_%1$s">
+						<p class="give_error give_notice give_%1$s" data-dismissible="%2$s" data-dismiss-interval="%3$d">
+							%4$s
+						</p>
+					</div>',
+					$notice_type,
+					give_clean( $notice_args['dismissible'] ),
+					absint( $notice_args['dismiss_interval'] ),
+					$message
+				);
+				break;
+
+			case 'manual':
+				$error = sprintf(
+					'<div class="give_notices give_errors" id="give_error_%1$s">
+						<p class="give_error give_notice give_%1$s" data-dismiss-type="%2$s">
+							%3$s
+						</p>
+						<button type="button" class="notice-dismiss give-notice-close"></button>
+					</div>',
+					$notice_type,
+					give_clean( $notice_args['dismiss_type'] ),
+					$message
+				);
+				break;
+
+			default:
+				break;
+		}
+
+		wp_enqueue_style( 'dashicons' );
```

Comment:
@Sidsector9 rather than using a dashicon can we use an SVG for the close icon?

Reply:
@DevinWalker Yes we can, just thought that using dashicons would make the front-end consistent with the admin screen. I'll make the changes.